### PR TITLE
Draft pull request for Feature/core, C# Business Core skeleton

### PR DESCRIPTION
**Pull request by jazd**
Significant fixes to Make targets
A start on Accounting functions
A start on Bills and Cargo (Invoice and Item)
C# Core skeleton with Book server-side function

Sample uses of Business C# core are in CSharp/SchemaVersion/Program.cs

### CODE REVIEW

**Comment by jazd** [comment]
Sample uses of Business C# core are in CSharp/SchemaVersion/Program.cs

File: CSharp/Core/ServerSide/Function.cs [hunk 1]
```diff
@@ -58,5 +54,9 @@ public partial class Function
 			database.Connection.Close();
 			return result;
 		}
+		private const string BookBalanceSQL = @"
+SELECT book, entry, account, nameid, name, rightSide, type AS typeId, typename AS type, debit, credit
+FROM BookBalance(@book, @amount)
+";
 	}
 }
```
Comment (JacobPenland) [inline CSharp/Core/ServerSide/Function.cs:62]: Carriage return

File: CSharp/SchemaVersion/packages.config [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-  <package id="System.Runtime.CompilerServices.Unsafe" version="4.6.0" targetFramework="net47" />
-  <package id="System.Text.Encodings.Web" version="4.6.0" targetFramework="net47" />
-  <package id="System.Text.Json" version="4.6.0" targetFramework="net47" />
-  <package id="System.Threading.Tasks.Extensions" version="4.5.3" targetFramework="net47" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.7.1" targetFramework="net47" />
+  <package id="System.Text.Encodings.Web" version="4.7.0" targetFramework="net47" />
+  <package id="System.Text.Json" version="4.7.1" targetFramework="net47" />
+  <package id="System.Threading.Tasks.Extensions" version="4.5.4" targetFramework="net47" />
   <package id="System.ValueTuple" version="4.5.0" targetFramework="net47" />
 </packages>
```
Comment (JacobPenland) [inline CSharp/SchemaVersion/packages.config:21]: Carriage return

Comment (jazd) [reply CSharp/SchemaVersion/packages.config:21]: @JacobPenland  Thanks for pointing those out, you know how much I dislike them :-)
I will make the edits in the develop branch.

**Pull request closed without merging**